channels/tr.m3u

```diff
@@ -1,4 +1,6 @@
 #EXTM3U x-tvg-url="http://195.154.221.171/epg/guideturkey.xml.gz"
+#EXTINF:-1 tvg-logo="https://i.imgur.com/aJDmVr3.png",Teve 2
+https://teve2.dogannet.tv/S2/HLS_LIVE/teve2np/track_4_1000/playlist.m3u8
```

Comment:
Apparently this stream is geo-blocked. It works on streemtest, but VLC gives me an error.